azure-pipelines.yml

```diff
@@ -0,0 +1,170 @@
+# Ruby
+# Package your Ruby project.
+# Add steps that install rails, analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/ruby
+
+name: '$(SourceBranchName)-$(Build.SourceVersion) Build-$(Build.BuildId)'
+
+trigger:
+  batch: true
+  branches:
+    include:
+      - develop
+
+pr:
+  autoCancel: true
+  branches:
+    include:
+      - develop
+      - feature/*
+
+variables:
+  application.name: 'et-ccd-export'
+  azure.subscription.endpoint: 'hmcts-pet'
+  manual.branch.prefix: 'develop'
+
+jobs:
+- job: TestApplication
+
+  pool:
+    vmImage: 'Ubuntu-16.04'
+  
+  steps:
+  - task: UseRubyVersion@0
+    inputs:
+      versionSpec: '>= 2.5'
+
+  - script: |
+      export ENABLE_COVERAGE=true
+      export DB_USERNAME=postgres
+      export DB_PORT=5433
+      export CC_TEST_REPORTER=935a4195bd35a993666acc398a01a53daf4e7f11635e90a51c8f30b44cbe1ec9
+      sudo apt-get update
+      sudo apt-get install libpq-dev mcrypt libmcrypt-dev pdftk
```

Comment:
Dont need any of these apart from maybe libpq-dev in case the db gem is in there